### DISCUSSION TITLE
Can't change link color

### DISCUSSION THREAD

**Discussion by lapisdecor** [zim-desktop-wiki/zim-desktop-wiki#2568] [General]
I'm currently using Ubuntu and I installed the zim flatpak.
On the FAQ I see I can change the link color (which is illegible in my case because I'm using dark theme) by copying "/usr/share/zim/style.conf" but I cant find that file on my system. Where is it located when using the flatpak and where should I copy it to?
Thanks.

**Comment by introt** [comment]
It should be picked up from `$HOME/.var/app/org.zim_wiki.Zim/config/zim/style.conf`, you can find the default here: https://raw.githubusercontent.com/zim-desktop-wiki/zim-desktop-wiki/develop/data/style.conf

**Reply by lapisdecor** [reply]
Thanks! It worked :+1:
